src/component/dataZoom/AxisProxy.ts

```diff
@@ -271,7 +271,22 @@ class AxisProxy {
         const valueWindow = this._valueWindow;
 
         if (filterMode === 'none') {
-            return;
+          each(seriesModels, function (seriesModel) {
+            if (seriesModel.subType === 'bar' || seriesModel.subType === 'scatter') {
```

Comment:
It may not be a good idea to check `subType` here because AxisProxy should have no idea of what series type is. These logic should probably be implemented in their own series code.